test/spec/core/image.spec.js

```diff
@@ -512,7 +514,8 @@ describe("Image", function() {
 
     };
 
-    var image = cartodb.Image(vizjson).size(400, 300);
+    var options = { auth_tokens: ["e900fe76cc3c1eed4fc018d027d82c8b0e59b2c484d1941954f34b4818a5d660"] };
```

Comment:
What is `e900fe76cc3c1eed4fc018d027d82c8b0e59b2c484d1941954f34b4818a5d660`?

Reply:
Do we need this variable?

Reply:
I just moved it from the vizjson stub, it's the token that will later be expected